README.md

```diff
@@ -239,6 +239,13 @@ as `deps/icu` (You'll have: `deps/icu/source/...`)
 > vcbuild full-icu
 ```
 
+### Testing
+
+To quickly test that io.js was built correctly from any of the steps above:
+```
```

Comment:
minor nit: add a space before the code block

also would be good to have the same code work on Windows and Unixes if you're in a position to check, if not then duplicate the block and for the Windows block use `>` instead of `$`